src/main/java/eu/europa/ted/eforms/sdk/component/SdkComponentFactory.java

```diff
@@ -126,22 +142,40 @@ protected <T> T getComponentImpl(String sdkVersion, final SdkComponentType compo
   }
 
   protected <T> T getComponentImpl(String sdkVersion, final SdkComponentType componentType,
-      final String qualifier, final Class<T> intf, Object... initArgs) throws InstantiationException {
+      final String qualifier, final Class<T> intf, Object... initArgs)
+      throws InstantiationException {
 
     String normalizedVersion = normalizeVersion(sdkVersion);
 
     ComponentSelector selector = new ComponentSelector(componentType, qualifier);
 
+    Map<ComponentSelector, SdkComponentDescriptor<?>> map =
+        Optional.ofNullable(componentsMap.get(normalizedVersion))
+            .orElseGet(Collections::emptyMap);
+
+    if (logger.isDebugEnabled()) {
+      logger.debug("selector componentType={}", selector.componentType);
+      logger.debug("selector qualifier={}", selector.qualifier);
+      logger.debug("normalized version={}", normalizedVersion);
+      for (Entry<ComponentSelector, SdkComponentDescriptor<?>> entry : map.entrySet()) {
+        logger.debug(
+            "entry key componentType={}, key qualifier={}, value={}",
```

Comment:
This message should be more explicit, for example:
"Available component for this version: componentType=[{}], qualifier=[{}]"

Reply:
Ok for both, I was seeing this too much from my perspective, yes the core is used by others...

Reply:
Message is now OK.